### DISCUSSION TITLE
C2O not receiving my orders once live

### DISCUSSION THREAD

**Discussion by ChipHosting** [AshleyRedman/WPClothes2Order#83] [Q&A] [answered]
Hi,

So I have tested the app and sent in C2O test mode to C2O, however now ive been marked as live at C2O they are not receiving my order. I have unticked the test options so it should be sending to C2O.  Looking at the order it sends to C2O but no response
![image](https://user-images.githubusercontent.com/26192155/201196830-b41d361b-39b8-4398-a34b-f41d10982f11.png)

![image](https://user-images.githubusercontent.com/26192155/201197312-46727cb6-224a-4b12-9356-d256e0f52176.png)

![image](https://user-images.githubusercontent.com/26192155/201197396-da1366e3-f203-455c-ae2d-547ed4992738.png)

UPDATE:
Speaking to C2O my order is still being sent in test mode. Can you assist? See below i dont have it set to test mode:
![image](https://user-images.githubusercontent.com/26192155/201308160-0832261b-c887-42cd-bcb5-d33e8522e091.png)

![image](https://user-images.githubusercontent.com/26192155/201308243-7356351e-9fee-4497-acba-d424a8b5888a.png)

![image](https://user-images.githubusercontent.com/26192155/201308374-bc4f7dfa-364f-4736-b3e2-4166712bfd50.png)

Thanks,

Adam

**Comment by AshleyRedman** [comment, marked as answer]
Hey @adamcook102 thanks for letting me know. I will look to release a hotfix this weekend which should handle this. As you say it looks like test mode isn't being turned off when you toggle it off.

**Reply by AshleyRedman** [reply]
Hi @adamcook102 I have released a hotfix for this now in the latest 1.0.3 version. Available to download here: https://wordpress.org/plugins/wpclothes2order/

So uploading the new version should fix the issue :)
